scripts/imgtool/main.py

```diff
@@ -288,8 +288,10 @@ def convert(self, value, param, ctx):
               help='Specify the value of security counter. Use the `auto` '
               'keyword to automatically generate it from the image version.')
 @click.option('-v', '--version', callback=validate_version,  required=True)
-@click.option('--align', type=click.Choice(['1', '2', '4', '8']),
+@click.option('--align', type=click.Choice(['1', '2', '4', '8', '16', '32']),
               required=True)
+@click.option('--max-align', type=click.Choice(['1', '2', '4', '8', '16', '32']),
```

Comment:
Eventually once other tools are updated this extra parameter could be removed and the new MAGIC with max-align could be used all the time...

Reply:
I wonder whether not better to only extend --align allowed values range.
New magic will then apply only for 16 and 32, so answer is no,
i think we need to explain what is the difference between --align and - max-align to the user

And: Should `--align value` >= `--max_align value` ?